File: exercicios/1047/1047.py
x = input().split()

h_ini = int(x[0])
m_ini = int(x[1])
h_fim = int(x[2])
m_fim = int(x[3])

dia = 24
hora = 60

if(h_ini == h_fim) and (m_ini == m_fim):
    m = 0
    print('O JOGO DUROU {:.0F}'.format(dia),'HORA(S) E {:.0F}'.format(m),'MINUTO(S)')

elif(h_ini <= h_fim) and (m_ini <= m_fim):
    h = (h_fim - h_ini)
    m = (m_fim - m_ini)
    print('O JOGO DUROU {:.0F}'.format(h),'HORA(S) E {:.0F}'.format(m),'MINUTO(S)')

elif(h_ini <= h_fim) and m_ini >= m_fim:
    if(h_fim - h_ini) > 0 and m_ini >= m_fim:
        h = 0
        m =   hora - (m_ini - m_fim)
        print('O JOGO DUROU {:.0F}'.format(h),'HORA(S) E {:.0F}'.format(m),'MINUTO(S)')
    
    elif (h_fim - h_ini) == 0 and (m_ini >= m_fim):
        h = 23
        m = 60 - (m_ini - m_fim)
        print('O JOGO DUROU {:.0F}'.format(h),'HORA(S) E {:.0F}'.format(m),'MINUTO(S)')

elif(h_ini >= h_fim) and m_ini >= m_fim:
    if(h_fim - h_ini) > 0 and m_ini >= m_fim:
        h = 0
        m =   hora - (m_ini - m_fim)
        print('O JOGO DUROU {:.0F}'.format(h),'HORA(S) E {:.0F}'.format(m),'MINUTO(S)')

File: exercicios/JavaScript/1047/1047.js
var input = require('fs').readFileSync('stdin','utf8');
var lines = input.split('\n');

var line1 = lines.shift().split(' ');

var hi = parseInt(line1[0])
var mi = parseInt(line1[1])
var hf = parseInt(line1[2])
var mf = parseInt(line1[3])

var h = 0, m = 0;

if(hi < hf){
    h = hf - hi;
    if(mi < mf){
        m = mf - mi;
    }else if(mi > mf){
        h = h - 1;
        m = (60 - mi) + mf;
    }else if(mi == mf){
        m = 0;
    }
}
if(hi > hf){
    h = (24 - hi) + hf;
    if(mi < mf){
        m = mf - mi;
    }else if(mi > mf){
        h = h - 1;
        m = (60 - mi) + mf;
    }else if(mi == mf){
        m = 0;
    }
}
if(hi == hf){
    if(mi < mf){
        m = mf - mi;
        h = 0;
    }else if(mi > mf){
        m = (60 - mi ) + mf;
        h = 23;
    }else if(mi == mf){
        h = 24;
        m = 0;
    }
}

console.log("O JOGO DUROU " + h + " HORA(S) E " + m + " MINUTO(S)");
